static/app/components/timeSince.tsx

```diff
@@ -30,6 +30,10 @@ interface Props extends React.TimeHTMLAttributes<HTMLTimeElement> {
    * that
    */
   disabledAbsoluteTooltip?: boolean;
+  /**
+   * Tooltip text to be hoverable when isTooltipHoverable is true
+   */
+  isTooltipHoverable?:boolean;
```

Comment:
I am on my work machine and didnt wanna do the linter setup to mess up my current settings, sorry for this linter miss. 
will keep this in mind from now on if I add anything more , thanks for this.

Reply:
That makes sense. Not a problem for me tbh. We do have a bot that comes by, but doesn't work on forks because it needs privileged credentials to make commits. I'm running around a bit more than usual today but it'll get in and deployed.